src/sql/workbench/contrib/notebook/browser/find/notebookFindModel.ts

```diff
@@ -59,7 +60,7 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 	private _decorations: { [decorationId: string]: NotebookIntervalNode; };
 	//#endregion
 
-	constructor(private _notebookModel: INotebookModel) {
+	constructor(private _notebookModel: INotebookModel | undefined) {
```

Comment:
It seems weird to me that we'd ever have undefined here. Is that actually a valid scenario?

Reply:
+1. Are we (unintentionally) doing this anywhere? If not, I'm not sure that we need to make this change.

Reply:
reverted